TADashboard/src/main/java/page_objects/DataProfilesPage.java

```diff
@@ -0,0 +1,26 @@
+package page_objects;
+
+import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Link;
+import utils.common.Constants;
+
+public class DataProfilesPage {
+    private CheckBox inFrontCheckBox = new CheckBox("xpath=//table[@class='GridView']/tbody/tr/td[text()='%s']/preceding-sibling::td/input");
+    private Link dataProfileLink = new Link("xpath=//table[@class='GridView']/tbody/tr/td[text()='%s']");
```

Comment:
why do you keep having inconsistent control names?

Reply:
Ok, I fixed it all.